mylyn.context/org.eclipse.mylyn.context.tasks.ui/src/org/eclipse/mylyn/internal/context/tasks/ui/editors/ContextEditorFormPage.java

```diff
@@ -231,20 +231,27 @@ public void dispose() {
 		}
 	}
 
+	private int setNewMaxWith(int oldValue, Control imageControl, Control linkControl) {
```

Comment:
Would you like to name arguments to be single word, like `existing`, 'image', `link`